tornado/autoreload.py

```diff
@@ -270,7 +272,15 @@ def main():
     can catch import-time problems like syntax errors that would otherwise
     prevent the script from reaching its call to `wait`.
     """
+    # Remember that we were launched with autoreload as main.
+    # The main module can be tricky; set the variables both in our globals
+    # (which may be __main__) and the real importable version.
```

Comment:
Is this true with the use of `tornado.autoreload.main` below? I think we only need to import `tornado.autoreload` in one of these two places. This one could just assign to the module globals.